doc/devguide/codebase/code-style.rst

```diff
@@ -30,6 +183,19 @@ We use 4 space indentation.
             return TM_ECODE_FAILED;
         }
 
+Note, use 8 space indentation when wrapping function parameters.
```

Comment:
clang only support on "Continuation Indent", hence function wrapping should be indented by 8 as well